deploy/csv/redhat/1.4.0/manifests/perconaxtradbclusters.pxc.percona.com.crd.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: perconaxtradbclusters.pxc.percona.com
+spec:
+  group: pxc.percona.com
+  names:
+    kind: PerconaXtraDBCluster
+    listKind: PerconaXtraDBClusterList
+    plural: perconaxtradbclusters
+    singular: perconaxtradbcluster
+    shortNames:
+      - pxc
+      - pxcs
+  scope: Namespaced
+  version: v1-4-0
+  versions:
+    - name: v1-4-0
```

Comment:
versions seem unordered, which is hard to maintain, same goes for other files

Reply:
done